src/RoutingDuplexMiddleware.purs

```diff
@@ -69,13 +69,13 @@ execute req env = unsafePerformEffect $ do
         Default -> do
           req' <- Req.replyStatus (StatusCode 404) req
           pure $ stopResult req'
-        DefaultHandler handler -> 
-          let env' = updateEnv handler env
+        DefaultHandler handler ->
+          let env' = updateEnv handler
           in pure $ okResult req env'
         CowboyRouterFallback ->
           cowboyRouterExecute req env
 
-    updateEnv { mod, args } env = 
-      Map.insert (atom "handler") (unsafeToForeign mod) 
+    updateEnv { mod, args } =
```

Comment:
Kind of feel this is super wierd, a function that updates the original env, I'd really rather it just operated `Env -> Env` or had some name that didn't imply it was an update function

Reply:
Which is not to say the behaviour has changed in the current context, but I think it's harder to grok